core/lib/utils/duration_utils.dart

```diff
@@ -0,0 +1,89 @@
+class DurationUtils {
+
+  // this class is a local copy of the one found at
+  // https://github.com/desktop-dart/duration/blob/master/lib/src/parse/parse.dart#L6
+  static Duration parseDuration(String input, {String separator = ','}) {
```

Comment:
why we need this? how about using dependency directly?

Reply:
I remembered it was asked by @tddang-linagora

Reply:
We should minimize the using of third party library. If we only need a small function, let's copy it over.

Reply:
from my point of view, we should use 3rd party lib to get the support from the community, don't need effort in maintaining it.

Reply:
Tend to agree with @hoangdat .

Why maintain this code if we have it from a healthy package (size, community, etc..)

Reply:
i'll drop the commit `removed package duration usages` when squashing then (if ok for you @tddang-linagora)

Reply:
done !